packages/@sanity/cli/src/commands/functions/testFunctionsCommand.ts

```diff
@@ -1,76 +1,96 @@
 import {type CliCommandDefinition} from '../../types'
 
 const helpText = `
+Arguments
+  <name> The name of the Sanity Function
+
 Options
   --data <data> Data to send to the function
   --file <file> Read data from file and send to the function
-  --name <name> The name of your Sanity Function
   --timeout <timeout> Execution timeout value in seconds
+  --api <version> Sanity API Version to use
+  --dataset <dataset> The Sanity dataset to use
+  --project-id <id> Sanity Project ID to use
+
 
 Examples
   # Test function passing event data on command line
-  sanity functions test --name echo --data '{ "id": 1 }'
+  sanity functions test echo --data '{ "id": 1 }'
 
   # Test function passing event data via a file
-  sanity functions test -name echo --file 'payload.json'
+  sanity functions test echo --file 'payload.json'
 
   # Test function passing event data on command line and cap execution time to 60 seconds
-  sanity functions test -name echo --data '{ "id": 1 }' --timeout 60
+  sanity functions test echo --data '{ "id": 1 }' --timeout 60
 `
 
 const defaultFlags = {
-  data: undefined,
-  file: undefined,
-  name: '',
-  timeout: 5, // seconds
+  'data': undefined,
+  'file': undefined,
+  'timeout': 5, // seconds
```

Comment:
Fack, we should update this to 10 seconds which is the new default. Probably should do that in runtime-cli as well.